res/values/strings_en.arb

```diff
@@ -679,5 +679,7 @@
   "tor_only": "Tor only",
   "unmatched_currencies": "Your current wallet's currency does not match that of the scanned QR",
   "contact_list_contacts": "Contacts",
-  "contact_list_wallets": "My Wallets"
+  "contact_list_wallets": "My Wallets",
+  "do_not_send": "Don't send",
+  "error_dialog_content": "Oops, we got some error.\n\nPlease send crash report to our support team to make the application better."
```

Comment:
Please send the crash report to our support team to make the application better.

(add `the`)